src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionHotlineViewController.swift

```diff
@@ -17,7 +17,24 @@
 
 import UIKit
 
-class ExposureSubmissionHotlineViewController: DynamicTableViewController, ENANavigationControllerWithFooterChild {
+class ExposureSubmissionHotlineViewController: DynamicTableViewController, ENANavigationControllerWithFooterChild, ExposureSubmissionCoordinatorViewController {
+
+	// MARK: - Attributes.
+
+	var coordinator: ExposureSubmissionCoordinator?
```

Comment:
is the conformance to the ExposureSubmissionCoordinatorViewController necessary? you pass it anyway via the constructor. is it necessary to publicly expose it?

Reply:
good catch, this can be removed entirely since i refactored the code to inject the coordinator in the constructor